interview-questions.md

```diff
@@ -10,42 +10,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Your answer:
 
-Researched answer:
+Researched answer: Hashes in Ruby consist of key:value pairs. It is similar to an array, yet different because the indexing is not static as it is with arrays. In hashes, indexing is done with arbitrary keys of an object type. You create a hash by using the arrow function to assign a key to a value key => value and separating each key value pair with a comma. The entire has should be enclosed in curly braces. 
+
+{ "jardin" => "garden", "fenetre" => "window", "avion" => "airplane" }
 
 2. What is a gem?
 
-Your answer:
+Your answer: A gem refers to a block of code used in Ruby.
 
-Researched answer:
+Researched answer: A gem is a software package used with Ruby that is used to pass on applications or libraries. Gems can be installed via the command line. Rubygems.org has a great deal of ruby gems available. ActiveRecord is an example of a Ruby gem that I have used before.  
 
 3. What is Ruby on Rails?
 
-Your answer:
+Your answer: 
 
-Researched answer:
+Researched answer: Ruby on Rails is a web application framework written in Ruby. It is an MVC (Model/View/Controller) framework that provides the structures for a database, web service and web pages. Some well-known companies utilizing Ruby on Rails include AirBnB, Square, GitHub and Shopify. 
 
 4. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: A database is an organized collection of data. A relational database is a database in which we understand how data relates to other data. 
+
+Researched answer: A relational database is a model created in the 1970's by E.F. Codd from IBM. It allows tables to be related to one another using a common attribute. In this way we can understand how data is related to other data and use that knowledge to call on it effectively. For example, a library with an online lending system would have a table that contained data for the library user, such as Library ID, name, address, and preferred library location. The system would also have a table containing requested items which would include data such as Media ID, Library ID, current location of media, and Date of Delivery to preferred location. These tables are now related due to the shared Library ID, which enables you to utilize a foreign key to connect the tables. Some examples of more complicated relational databases are Cloud SQL and AlloyDB.
```

Comment:
good examples. I would say add more clarity on how that foreign key would appear also include the model association